### PR TITLE
Update _config.yml

### PR DESCRIPTION
**Pull request by jalfonze**

### CODE REVIEW

**Comment by jankdc** [comment]
Deploy preview processing.

Built with commit 73f4ccb9f5449e6e71b6dd90d46037f28113a349

https://app.netlify.com/sites/elegant-meitner-4ff78c/deploys/5a23534281987611c6955129

File: _config.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ default_img:
 
 # Profile settings
 name: Justin Alfonso
-profile_img: https://i.lensdump.com/i/dwuc3.jpg7
+profile_img: ![dwuc3.jpg](https://i.lensdump.com/i/dwuc3.jpg)
```
Comment (jankdc) [inline _config.yml]: this should just be https://i.lensdump.com/i/dwuc3.jpg

**Comment by jalfonze** [comment]
can't get the picture right on the footer, tried so many things as you can see. pls help.

**Comment by jankdc** [comment]
seems to be working now. merging.

**Pull request merged**